docs/modules/ROOT/pages/troubleshooting/index.adoc

```diff
@@ -31,3 +31,16 @@ spec:
             requests:
               storage: 1Gi  # increase accordingly
 ----
+
+
+== Pipeline Run Times Out ==
```

Comment:
I don't think that we use the trailing `==`s

Reply:
Fixed.